tests/baselines/reference/completionsCommentsClass.baseline

```diff
@@ -0,0 +1,3755 @@
+[
+  {
+    "marker": {
+      "fileName": "/tests/cases/fourslash/completionsCommentsClass.ts",
+      "position": 599
```

Comment:
It would be good to have marker name or line and coloumn info as well.

Reply:
Name is quite easy to add, so I did that. I don't know of an easy way to convert positions back to line/columns, though.